### PR TITLE
Update `Response` inline comments

### PR DESCRIPTION
**Pull request by 0xTim**

### CODE REVIEW

File: Sources/Vapor/Response/Response.swift [hunk 1]
```diff
@@ -22,9 +22,9 @@ public final class Response: CustomStringConvertible {
...
     public var headers: HTTPHeaders
     
-    /// The `HTTPBody`. Updating this property will also update the associated transport headers.
+    /// The `Body`. Updating this property will also update the associated transport headers.
     ///
-    ///     httpRes.body = HTTPBody(string: "Hello, world!")
+    ///     res.body = Response.Body(string: "Hello, world!")
     ///
     /// Also be sure to set this message's `contentType` property to a `MediaType` that correctly
     /// represents the `HTTPBody`.
```
Comment (stevapple) [inline Sources/Vapor/Response/Response.swift:30]:
```suggestion
    /// represents the `Body`.
```

File: Sources/Vapor/Response/ResponseCodable.swift [hunk 2]
```diff
@@ -2,22 +2,22 @@
...
 /// Types that conform to this protocol can be returned in route closures.
 public protocol ResponseEncodable {
-    /// Encodes an instance of `Self` to a `HTTPResponse`.
+    /// Encodes an instance of `Self` to a `Response`.
     ///
     /// - parameters:
-    ///     - for: The `HTTPRequest` associated with this `HTTPResponse`.
-    /// - returns: An `HTTPResponse`.
+    ///     - for: The `Request` associated with this `Response`.
+    /// - returns: An `Response`.
```
Comment (stevapple) [inline Sources/Vapor/Response/ResponseCodable.swift:9]:
```suggestion
    /// - returns: A `Response`.
```

File: Sources/Vapor/Response/Response.swift [hunk 3]
```diff
@@ -43,7 +43,7 @@ public final class Response: CustomStringConvertible {
 
     public var storage: Storage
     
-    /// Get and set `HTTPCookies` for this `HTTPResponse`
+    /// Get and set `HTTPCookies` for this `Response`
```
Comment (stevapple) [inline Sources/Vapor/Response/Response.swift:46]:
```suggestion
    /// Get and set `HTTPCookies` for this `Response`.
```

**Review by stevapple — CHANGES_REQUESTED** [review]

**Review by Joannis — APPROVED** [review]
As soon as @stevapple 's comments are resolved, I'm good 👍

**Review by stevapple — APPROVED** [review]
LGTM now👍

**Comment by 0xTim** [comment]
No code changes and not critical docs, so we'll roll this out in the next release

**Pull request merged**